js/components/user/list.vue

```diff
@@ -58,6 +58,13 @@ export default {
                 align: 'center',
                 type: 'metric',
                 width: 95
+            }, {
+                label: this._('Roles'),
+                key: 'roles',
+                sort: 'roles',
+                align: 'center',
+                type: 'metric',
```

Comment:
I agree it is not a metric.
But it is not really a label, since there can be, for one user, several roles.
Is there something similar in the code, for tags or badges for instance ?